sycl/test-e2e/BFloat16/bfloat16_builtins.cpp

```diff
@@ -286,20 +286,6 @@ int main() {
   TEST_BUILTIN_1(log, sycl::ext::oneapi::bfloat16);
   TEST_BUILTIN_1(log2, sycl::ext::oneapi::bfloat16);
   TEST_BUILTIN_1(log10, sycl::ext::oneapi::bfloat16);
-}
-
-TEST_BUILTIN_1(fabs, bfloat16);
```

Comment:
As this was removed in the latest commit, do we have tests for these functions somewhere else in the repo?

Reply:
Hi, @dm-vodopyanov 
I made a mistake when merging latest sycl branch into dev branch and introduced some redundant lines, this commit is to remove those redundant lines. Sorry for the confusion.
Thanks very much.